cli/azd/resources/starter/infra/main.bicep

```diff
@@ -0,0 +1,54 @@
+targetScope = 'subscription'
+
+@minLength(1)
+@maxLength(64)
+@description('Name of the the environment which is used to generate a short unique hash used in all resources.')
+param environmentName string
+
+@minLength(1)
+@description('Primary location for all resources')
+param location string
+
+// Optional parameters to override the default azd resource naming conventions.
+// Add the following to main.parameters.json to provide values:
+// "resourceGroupName": {
+//      "value": "myGroupName"
+// }
+param resourceGroupName string = ''
+param keyVaultName string = ''
+
+var abbrs = loadJsonContent('./abbreviations.json')
+var resourceToken = toLower(uniqueString(subscription().id, environmentName, location))
+// Tag the resource group with the current environment.
+var tags = { 'azd-env-name': environmentName }
+
+// Organize resources in a resource group
+resource rg 'Microsoft.Resources/resourceGroups@2021-04-01' = {
+  name: !empty(resourceGroupName) ? resourceGroupName : '${abbrs.resourcesResourceGroups}${environmentName}'
+  location: location
+  tags: tags
+}
+
+// Store secrets in a keyvault, using a bicep module. Remove if unused.
```

Comment:
I recommend we keep it simple and remove keyvault. abbreviations.json is fine but we should include comments to say what is it for, why and how to replace with own json file if they have their own conventions.

Reply:
Yep. Arrived at that conclusion myself based on how different it was from our core bicep modules.